client/app/bundles/course/statistics/pages/StatisticsIndex/index.jsx

```diff
@@ -0,0 +1,137 @@
+import PropTypes from 'prop-types';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { defineMessages, injectIntl, intlShape } from 'react-intl';
+
+import NotificationBar from 'lib/components/NotificationBar';
```

Comment:
Would you like to use react toast instead? We can then remove dependency on redux store for notification related UI.